src/mono/msbuild/common/LibraryBuilder.targets

```diff
@@ -0,0 +1,34 @@
+<Project>
+  <UsingTask TaskName="LibraryBuilderTask" 
+             AssemblyFile="$(LibraryBuilderTasksAssemblyPath)" />
+
+  <Target Name="_BuildNativeLibrary"
+          Condition="'$(_IsLibraryMode)' == 'true' and '$(RunAOTCompilation)' == 'true'">
+
+    <PropertyGroup>
+      <_IsSharedLibrary>false</_IsSharedLibrary>
+      <_IsSharedLibrary Condition="'$(NativeLib)' == 'shared'">true</_IsSharedLibrary>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_ExtraLibrarySources Include="$(_AotModuleTablePath)" />
+      <_ExtraLinkerArgs Include="@(_CommonLinkerArgs)" />
+    </ItemGroup>
+
+    <LibraryBuilderTask
+      Assemblies="@(AppAssembliesInternal)"
+      ExtraLinkerArguments="@(_ExtraLinkerArgs)"
+      ExtraSources="@(_ExtraLibrarySources)"
+      ExtraSymbolsToExport="@(_RuntimeSymbolsToKeep)"
```

Comment:
Since the `_RuntimeSymbolsToKeep` ItemGroups are no longer declared because `DirectPInvokes` and `DirectPInvokeLists` are being passed to the `MonoAOTCompiler` Task in AndroidApp.targets, do we need `ExtraSymbolsToExport`?

Reply:
I wondered about that.  Probably not to be honest.